CONTRIBUTING.md

```diff
@@ -436,13 +436,35 @@ and follow below steps to add a rule for azure rest api specs.
 - add a doc for the rule in 'docs', the rule doc file name is following kebab-case, contains following content:
 
 ```md
-# RuleName
+# <add RuleName>
+
+## Category
+
+ARM Error
```

Comment:
this needs to be generic place holder
could be ARM error or datat place or both right?

Reply:
We can update this to "ARM/SDK Warning", "ARM/SDK Error" but not considered Data Plane rules. We are checking linter rules only on "azure-rest-api-specs", so I mentioned only "ARM OpenAPI(swagger) specs" 
@rkmanda can you please confirm if I need to update these standards to satisfy Data Plane rules as well?

Reply:
lets provide all possible values here

Reply:
Added all values